### DISCUSSION TITLE
搜索优化：添加默认搜索路径

### DISCUSSION THREAD

**Discussion by iris2jane** [obsidianzh/forum#48] [Ideas想法]
希望可以将常用的搜索路径添加为默认
<img width="519" alt="image" src="https://user-images.githubusercontent.com/36763930/105797445-f4cc6300-5fca-11eb-903a-4bd4f41574e8.png">

就像使用这款划词插件能够切换不同的词典一样👇
<img width="460" alt="image" src="https://user-images.githubusercontent.com/36763930/105797634-5391dc80-5fcb-11eb-9d28-5359f679fdc1.png">

**Comment by makerjackie** [comment]
可以提在 forum.obsidian.md 官方英文论坛中，这里开发者可能看不到

**Reply by iris2jane** [reply]
嗯我待会在discord at下，懒得发英文了……
